package.json

```diff
@@ -67,7 +67,8 @@
     "extends": "./node_modules/kcd-scripts/eslint.js",
     "rules": {
       "import/extensions": "off",
-      "@typescript-eslint/no-unsafe-assignment": "off"
+      "@typescript-eslint/no-unsafe-assignment": "off",
+      "max-lines-per-function": "off"
```

Comment:
Does this seem okay to you? This is the only reason validate is failing at the moment.

Reply:
Yes, that rule is super annoying and I should just disable it.